### PR TITLE
chore(demo): migrate documentation page `ComboBox` to standalone

### PR DESCRIPTION
**Pull request by ThomasClague**
Partially solves #6957 
Done: Migrate ComboBox component to standalone

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/combo-box/examples/1/index.ts [hunk 1]
```diff
@@ -25,13 +36,25 @@ const databaseMockData: readonly User[] = [
...
 
 @Component({
-    selector: 'tui-combo-box-example-1',
+    standalone: true,
+    imports: [
+        AsyncPipe,
+        TuiLetModule,
+        ReactiveFormsModule,
+        TuiComboBoxModule,
+        TuiDataListModule,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/combo-box/examples/1/index.ts:45]: Could we drop import of this module ? 🤔

File: projects/demo/src/modules/components/combo-box/index.ts [hunk 2]
```diff
@@ -28,16 +39,30 @@ class Account {
 }
 
 @Component({
-    selector: 'example-tui-combo-box',
-    templateUrl: './combo-box.template.html',
-    styleUrls: ['./combo-box.style.less'],
+    standalone: true,
+    imports: [
+        TuiDemoModule,
+        FormsModule,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/combo-box/index.ts:45]: Could we drop this import ?

File: projects/demo/src/modules/components/combo-box/index.ts [hunk 3]
```diff
@@ -28,16 +39,30 @@ class Account {
...
+        ReactiveFormsModule,
+        TuiComboBoxModule,
+        TuiDataListModule,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/combo-box/index.ts:48]: Could we drop it ?

**Review by nsbarsukov — APPROVED** [review]
Thanks ❤️

**Comment by nsbarsukov** [comment]
Rebase the branch, please

File: projects/demo/src/modules/components/combo-box/examples/1/index.ts (hunk 1 above)
Comment (ThomasClague) [reply projects/demo/src/modules/components/combo-box/examples/1/index.ts]: Removed

File: projects/demo/src/modules/components/combo-box/index.ts (hunk 2 above)
Comment (ThomasClague) [reply projects/demo/src/modules/components/combo-box/index.ts]: Removed

**Pull request merged**